app/assets/javascripts/app/components/comment_form/templates/comment_form.html.slim

```diff
@@ -2,12 +2,16 @@
   .TopBarLayout-header
     md-toolbar on-scroll="" on-scroll-anchor="#js-comment-content" scrolling-class="md-whiteframe-z1"
       md-content.md-padding layout="row" layout-align="space-between center"
-        md-button.md-primary.md-warn md-no-ink="" ng-click="toggle()" layout="row" layout-align="center center"
+        md-button.md-primary.md-warn md-no-ink="" ng-click="ctrl.toggle()" layout="row" layout-align="center center"
           .IconClose
           | hide
 
-        council-raised-button.Button.Button--secondary ng-click="create(comment)"
```

Comment:
For future reference: i believe that having all context made available to `create` method (or any controller method) via explicitly passing all required arguments instead of relying on the controller's scope is better. Mainly because it makes them more isolated, which in turn makes them easier to test